.rubocop.yml

```diff
@@ -125,3 +125,6 @@ Rails/Output:
 # Disabled due to a bug
 Rails/BulkChangeTable:
   Enabled: false
+
+Capybara/FeatureMethods:
+  Enabled: false
```

Comment:
what  do you mean in the commit message with fixing it properly? :confused:

Reply:
What I wrote in the commit message :smile: Currently it relies on the subfolder and there are several possibilities how to "fix" it which we all need to discuss first and then create a card in the backlog.

Reply:
what about a `FIX` comment with the reason here? :thinking:

Reply:
Actually we usually always include the reason when disabling a cop :smile: